src/utils.ts

```diff
@@ -253,7 +254,13 @@ export default abstract class BaseCommand extends Command {
     if (sync) {
       await sequelize.sync({ force: true })
     }
+
+    // Init store
     await initStore(sequelize)
+
+    // Run migration
+    await DbMigration.getInstance(path).up()
```

Comment:
Hmm can we wrap this in `if` with check if something needs to be migrated and if so inform the user about it with some "spinner"?

Reply:
I don't see the reason to warp it in If statement, as it's fine that we run migration every time. Instead, I just add the check if we actually need to migrate something

Reply:
Ok, that is also valid option 👍